dev/Repeater/ViewportManagerDownlevel.cpp

```diff
@@ -166,30 +171,33 @@ void ViewportManagerDownLevel::OnElementCleared(const winrt::UIElement& element)
 
 void ViewportManagerDownLevel::OnOwnerArranged()
 {
-    m_expectedViewportShift = {};
-
-    EnsureScrollers();
-
-    if (HasScrollers())
+    if (!m_managingViewportDisabled)
```

Comment:
Style: For this sort of check I think it's fine to do an early return rather than put a condition/indent around the whole method body. Although I admit to being swayed by the fact that that style would have made this review diff look smaller with fewer lines touched ;)